evm/v0.5/contracts/dev/PrepaidAggregator.sol

```diff
@@ -168,8 +168,15 @@ contract PrepaidAggregator is AggregatorInterface, Owned, WithdrawalInterface {
     onlyOwner()
     onlyEnabledAddress(_oracle)
   {
-    oracleCount -= 1;
     oracles[_oracle].endingRound = reportingRoundId;
+    uint32 oracleNum = oracleCount();
+    for (uint32 i = 0; i < oracleNum; i++) {
```

Comment:
If we wanted to make this a constant gas cost, we could store the oracle's position in the array on the OracleStatus. The downside is it would require using another word of storage, or trimming down the size of one of the other variables.

Reply:
Went with trimming down withdrawable since it has extra room (max supply of LINK is 1000000000000000000000000000 while max uint120 is 1329227995784915872903807060280344575). This did require making a new SafeMath120.sol contract.